modules/openapi-generator/src/main/resources/go-server/controller-api.mustache

```diff
@@ -90,16 +90,21 @@ func (c *{{classname}}Controller) {{nickname}}(w http.ResponseWriter, r *http.Re
 	}
 	{{/isInteger}}
 	{{#isBoolean}}
-	{{paramName}}, err := parseBoolParameter(query.Get("{{baseName}}"))
```

Comment:
Should `query.Get` be using baseName (e.g. `created-date`) instead ?

Reply:
You're absolutely right: fixed.